fvm/evm/emulator/emulator.go

```diff
@@ -46,6 +46,7 @@ func newConfig(ctx types.BlockContext) *Config {
 		WithExtraPrecompiles(ctx.ExtraPrecompiles),
 		WithGetBlockHashFunction(ctx.GetHashFunc),
 		WithRandom(&ctx.Random),
+		WithTransactionTracer(ctx.Tracer),
```

Comment:
The name Transaction Tracer might be a bit confusing, I would suggest we call it debugTraces.